modules/admin/con-rbac-config-permission-policies-admin.adoc

```diff
@@ -16,3 +16,21 @@ permission:
       users:
         - name: user:default/joeuser
 ----
+
+The permission policy role (`role:default/rbac_admin`) is a default role in {product-short} and includes some permissions upon creation, such as creating, reading, updating, and deleting permission policies/roles, as well as reading from the catalog.
+
+If the default permissions are not adequate for your requirements, you can define a new administrator role tailored to your requirements using relevant permission policies. Alternatively, you can use the optional super user admin configuration value, which grants unrestricted permissions across {product-short}.
```

Comment:
Is `super user admin` a value that needs to be formatted as such? `superUsers` is the actual value, then lets replace with the following. 

```suggestion
If the default permissions are not adequate for your requirements, you can define a new administrator role tailored to your requirements using relevant permission policies. Alternatively, you can use the optional `superUsers` configuration value, which grants unrestricted permissions across {product-short}.
```